st2common/st2common/models/db/liveaction.py

```diff
@@ -75,6 +75,7 @@ class LiveActionDB(stormbase.StormFoundationDB):
         default={},
         help_text='Information about the runner which executed this live action (hostname, pid).')
     notify = me.EmbeddedDocumentField(NotificationSchema)
+    log = me.ListField(field=me.DictField())
```

Comment:
I'm not a fan of that part, but the way we implemented execution update [[1]](https://github.com/StackStorm/st2/blob/b3b394fd45db38c1aa4cd02e833498c15e4752db/st2common/st2common/services/executions.py#L132-L138) we would have to do some guessing to figure out whether update should or should not add a log entry.

Remind me once again, why do we need liveactions? =)